README.zh-CN.md

```diff
@@ -423,6 +444,31 @@ export default {
 
 </details>
 
+### `emit` 选项
+
+<details>
+<summary>
+❌ TS: 在 <code>defineComponent</code> 选项中添加 <code>emit</code> 事件校验将不会有任何效果, 这仅为针对 Vue3 的选项对齐。
```

Comment:
Can you help update this as well? Thanks

Reply:
Sure :smile:

Reply:
Done!